object.js

```diff
@@ -16,14 +16,14 @@ var convert24HourTime = function(number){
   }
 };
 
-var simulateCookiesPerHour = function () {
+var simulateCustomersPerHour = function () {
   var hours = [];
   for(var i = 0; i < this.hourClosed - this.hourOpen; i++) {
     var customerCount = this.randomCustomerPerHour();
     var time = (i + 6);
     //console.log('hours: ' + convert24HourTime(time) + ', cookies: ' + customerCount);
     hours[i] = customerCount * this.aveCookieSalePerCustomer;
-    this.cookiesPerHour = hours;
+    this.customersPerHour = hours;
```

Comment:
I don't remember if this is a change we paired on, but the old names were correct. On the line above, we're multiplying customers per hour by cookies per customer, so this value is, in fact, cookies per hour.